client/reader/lib/intl-collator/index.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Wasn't 100% sure where this should live - it uses `state` so it's like a selector, but it returns a `Intl.Collator` object.

This functionality will be reused to sort other Reader sidebars (tags, feeds) so I wanted to extract it.

Reply:
Could be a simple selector called `getSortLocales` instead?

Reply:
I'd make it a selector inside current-user/selectors